src/composables/useGetMessages.ts

```diff
@@ -144,6 +146,7 @@ export function useGetMessagesProvider() {
 			}
 			if (oldToken && oldToken !== newToken) {
 				store.dispatch('cancelPollNewMessages', { requestId: oldToken })
+				stopChatRelay()
```

Comment:
That should accompany all 5 of calls to `store.dispatch('cancelPollNewMessages')`